posts/2022/xarray-groupby-vs-geocat-climatology.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<p>I'm skeptical of a 3C difference. Is it still that different if you use a <code>groupby("time.season")</code> as above? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>No it's not. Now that I've switched to using <code>groupby("time.season")</code> the difference it +/- 0.1 C. This is much smaller and more like what I expected. </p>